overlord/assertstate/assertmgr_test.go

```diff
@@ -933,3 +943,57 @@ func (s *assertMgrSuite) TestSnapDeclaration(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(snapDecl.SnapName(), Equals, "foo")
 }
+
+func (s *assertMgrSuite) TestAutoAliases(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	// have a declaration in the system db
+	err := assertstate.Add(s.state, s.storeSigning.StoreAccountKey(""))
+	c.Assert(err, IsNil)
+	err = assertstate.Add(s.state, s.dev1Acct)
+	c.Assert(err, IsNil)
+	snapDeclFoo := s.snapDecl(c, "foo", nil)
```

Comment:
Might be easier to read if moved  closer to the `// empty list` part of the code where this is used (just like it is done with the `some aliases` part of the code).